mcu-interface/examples/log_messages.rs

```diff
@@ -19,19 +18,18 @@ async fn main() -> Result<()> {
         )
         .init();
 
-    let (msg_tx, msg_rx) = std::sync::mpsc::channel();
+    let (msg_tx, mut msg_rx) = tokio::sync::mpsc::channel(10);
```

Comment:
is there a risk to use an `UnboundedChannel` here as well? 🤔 
messages aren't big for the Jetson.

Reply:
I usually try to avoid unbounded channels whenever possible since usually you don't want memory to grow arbitrarily. I would have preferred to make the other one bounded too, but was worried about introducing some edge case that I didn't think about enough.

If we used bounded channels in both places, can you talk me through how we handle the case where the channel is full? Should we wait till it empties or warn and drop the message?

Reply:
I could also just change to an unbounded one for now and worry about making it bounded in a subsequent PR

Reply:
Switched to unbounded.